src/worldcereal/openeo/postprocess.py

```diff
@@ -32,6 +33,139 @@ def output_labels(self) -> list:
     def dependencies(self) -> list:
         return []
 
+    @classmethod
+    def majority_vote(
+        cls,
+        base_labels: xr.DataArray,
+        max_probabilities: xr.DataArray,
+        kernel_size: int,
+        conf_threshold: int,
+    ) -> xr.DataArray:
+        """
```

Comment:
can you reformat the docstring following the Numpy style format? Your autodocstring should be configured for that already I think?